packages/core/contracts/financial-templates/common/financial-product-libraries/KpiOptionsFinancialProductLibrary.sol

```diff
@@ -24,17 +24,17 @@ contract KpiOptionsFinancialProductLibrary is FinancialProductLibrary, Lockable
         nonReentrantView()
         returns (FixedPoint.Unsigned memory)
     {
-        // If price request is made before expiry, return 2. Thus we can keep the contract 100% collateralized with
-        // each token backed 1:2 by collateral currency. Post-expiry, leave unchanged.
+        // If price request is made before expiry, return 0.001. Thus we can keep the contract 100% collateralized with
+        // each token backed 1:0.001 by collateral currency. Post-expiry, leave unchanged.
         if (requestTime < ExpiringContractInterface(msg.sender).expirationTimestamp()) {
-            return FixedPoint.fromUnscaledUint(2);
+            return FixedPoint.fromUnsigned(1000000000000000);
```

Comment:
Looks right

Reply:
I think it was wrong and should have been `FixedPoint.Unsigned(1000000000000000)` instead